src/guiapi/src/gui.cpp

```diff
@@ -82,7 +84,7 @@ void gui_loop(void) {
             } else {
                 capturedWin->WindowEvent(capturedWin, WINDOW_EVENT_ENC_DN, (void *)-diff);
             }
-            gui_reset_menu_timer();
+            Screens::Access()->ResetTimeout();
```

Comment:
does Screen::Access always return a valid pointer?

Reply:
Yes, this is safe.

Reply:
Screen::Access calls bsod in case of invalid pointer